### DISCUSSION TITLE
Iframe

### DISCUSSION THREAD

**Discussion by keesfluitman** [chrede88/qubt#134] [General]
Hey, thanks for the nice and clean template. Really great work.
Now im trying to get an iframe integrated of an immich album i have. With setting width and height to 90vw, it works on mobile, but on desktop, the size is expanding whenever i use relative sizes. 
Is this something i could succesfully do? 

I fixed it like this now temporarily. Im guessing i could also create my own partial of course. but i was wondering if you had sth built in.
```
---
title: "Fotos"
layout: single
---

<iframe id="immich-album" src="iframelink" frameborder="0"></iframe>

<style>
#immich-album {
  width: 95vw;
  height: 90vw;
}

@media (min-width: 768px) { /* Adjust this breakpoint as needed */
  #immich-album {
    width: 800px;
    height: 600px;
  }
}
</style>
```

**Comment by chrede88** [comment]
I don't have any code for what you're trying to do. The easiest/best way to achieve your goal is to modify the partial by adding a file on the same path in your own repo. I can give some pointers if you want?

**Comment by keesfluitman** [comment]
Thanks for the response!
what do you mean add a file on the same path in my own repo?

I just liked your template as a basic functionality, and was checking if I could somehow expand it a bit with modules. But I guess much won't work (like accordeon or search for instance) because you don't have the scss files available right?

**Reply by chrede88** [reply]
So Hugo looks for files in both the theme module (i.e. the theme) and in your site repo. Let's say you wanted to modify the `single.html` layout file. This can be found on this following path: `./layouts/_default/single.html`. If you copy this file and put it into your site repo on the same path, Hugo will use this file and not the one in the theme. This way, you can locally modify the theme.

The theme is very lightweight by design. It's build with TailwindCSS and a tiny bit of custom JavaScript. You can add your own css, if you want to overwrite or extent the current classes. You can find the css here: https://github.com/chrede88/qubt/blob/main/assets/css/compiled/main.css

If you add a file called `./assets/css/custom.css` to your own repo, this will be integrated into your site.

This is also true for JS. `./assets/js/custom.js`

**Comment by keesfluitman** [comment]
thanks. yea im able to do all that. But I was going through what limited tutorials there is for modules, and see if i could add modules to yours. But It depends on importing stuff into the scss file. Which your repo doesn't have. So I guess Ill just see what else I can do.

Otherwise great work. Use it for my soccer team to add news and tournaments andstuff. I just implemented some more functionality to call an API with a database, but I will prolly build my own app for that in the future, as gohugo's purpose is actually much more about static than it is dynamics.
